Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EndOfYearDataManager.swift

```diff
@@ -17,7 +17,7 @@ class EndOfYearDataManager {
                                            """
 
     private lazy var listenedEpisodesPreviousYear = """
-                                            lastPlaybackInteractionDate IS NOT NULL AND lastPlaybackInteractionDate BETWEEN strftime('%s', '2021-01-01') and strftime('%s', '"2023-01-01"')
+                                            lastPlaybackInteractionDate IS NOT NULL AND lastPlaybackInteractionDate BETWEEN strftime('%s', '2022-01-01') and strftime('%s', '2023-01-01')
```

Comment:
Do we still need this and the `listenedEpisodesThisYear` if the `listenedEpisodes` does the same thing?

Reply:
Hmm probably not.

I'll write that down and refactor it on a subsequent PR just to avoid any merge conflict.